TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/data/multiple_tuples_parsed.json

```diff
@@ -0,0 +1,4 @@
+{"sourceRequest": {"Request-URI": "/", "Method": "GET", "HTTP-Version": "HTTP/1.1", "Host": "capture-proxy:9200", "User-Agent": "python-requests/2.32.3", "Accept-Encoding": "gzip, deflate, zstd", "Accept": "*/*", "Connection": "keep-alive", "Authorization": "Basic YWRtaW46YWRtaW4=", "body": ""}, "sourceResponse": {"HTTP-Version": {"keepAliveDefault": true}, "Status-Code": 200, "Reason-Phrase": "OK", "response_time_ms": 374, "content-type": "application/json; charset=UTF-8", "content-length": "538", "body": {"name": "3cc068ad54eb", "cluster_name": "docker-cluster", "cluster_uuid": "PIAawDwSQKSRLtoaXC-oWg", "version": {"number": "7.10.2", "build_flavor": "oss", "build_type": "docker", "build_hash": "747e1cc71def077253878a59143c1f785afa92b9", "build_date": "2021-01-13T00:42:12.435326Z", "build_snapshot": false, "lucene_version": "8.7.0", "minimum_wire_compatibility_version": "6.8.0", "minimum_index_compatibility_version": "6.0.0-beta1"}, "tagline": "You Know, for Search"}}, "targetRequest": {"Request-URI": "/", "Method": "GET", "HTTP-Version": "HTTP/1.1", "Host": "opensearchtarget", "User-Agent": "python-requests/2.32.3", "Accept-Encoding": "gzip, deflate, zstd", "Accept": "*/*", "Connection": "keep-alive", "Authorization": "Basic YWRtaW46bXlTdHJvbmdQYXNzd29yZDEyMyE=", "body": ""}, "targetResponses": [{"HTTP-Version": {"keepAliveDefault": true}, "Status-Code": 200, "Reason-Phrase": "OK", "response_time_ms": 341, "content-type": "application/json; charset=UTF-8", "content-length": "568", "body": {"name": "e6b3a4370763", "cluster_name": "docker-cluster", "cluster_uuid": "88zlik7ORFuFT78zduradA", "version": {"distribution": "opensearch", "number": "2.15.0", "build_type": "tar", "build_hash": "61dbcd0795c9bfe9b81e5762175414bc38bbcadf", "build_date": "2024-06-20T03:27:32.562036890Z", "build_snapshot": false, "lucene_version": "9.10.0", "minimum_wire_compatibility_version": "7.10.0", "minimum_index_compatibility_version": "7.0.0"}, "tagline": "The OpenSearch Project: https://opensearch.org/"}}], "connectionId": "0242acfffe12000b-0000000a-00000003-4c7bb8f4aefa3189-c0544dbf.0", "numRequests": 1, "numErrors": 0}
```

Comment:
nit: consider pretty-formatting all of these, maybe one per file and put them into a directory structure.  Then you can cons them together with the python equivalent of (cat foo/*).
It's tough to figure out what the errors are in each one.  Being able to name each will let the developer know what the faults or examples are for too.